public/js/modules/start.js

```diff
@@ -90,11 +100,13 @@ app.onUpdate = function(time){
 
 app.pause = function(){
     if(state == 'GAME'){
+        music.pause() // Pause Game Music
```

Comment:
music.pause() isn't descriptive enough so you added `// Pause Game Music` to make sure the code reader understands right? :D

Reply:
Hi, yes I define the "music" object and added some comments when I invoke the methods